Apps/WebClient/test/functional/e2e/cypress/integration/pages/Login.js

```diff
@@ -4,10 +4,7 @@ describe("Login Page", () => {
     });
 
     it("Validate URL", () => {
-        cy.url().should(
-            "contains",
-            "https://dev.healthgateway.gov.bc.ca/login"
-        );
+        cy.url().should("include", "login");
```

Comment:
using regex in cy may be better here in case some redirections contain also login key word.

Reply:
cy.url inspects the landed page and so this test works against all environments and we shouldn't have login in any of our routes.